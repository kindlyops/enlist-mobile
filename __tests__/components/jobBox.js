import React from 'react'
import renderer from 'react-test-renderer'
import { fromJS } from 'immutable'
import moment from 'moment'
import JobBox from '../../app/components/jobs/box'

describe('Initial state', () => {
  test('basics', () => {
    let job = fromJS({
      title: 'Example Job',
      jobType: 'Full-time',
      remote: true,
      status: 'active'
    })

    const tree = renderer.create(<JobBox job={job} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
