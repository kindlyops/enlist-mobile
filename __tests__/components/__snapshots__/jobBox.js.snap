// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Initial state basics 1`] = `
<View
  accessible={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "borderColor": "#EEEEEE",
      "borderRadius": 4,
      "borderWidth": 1,
      "marginTop": 15,
      "opacity": 1,
      "padding": 15,
      "paddingLeft": 30,
      "paddingRight": 20,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#848A96",
          "borderRadius": 20,
          "height": 6,
          "left": 15,
          "position": "absolute",
          "top": 20,
          "width": 6,
        },
        Object {
          "backgroundColor": "#C1C7D4",
        },
      ]
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontWeight": "500",
      }
    }
  >
    Example Job
  </Text>
  <View
    style={
      Object {
        "flexDirection": "row",
        "marginTop": 2,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#848A96",
          },
          Object {
            "fontSize": 13,
          },
        ]
      }
    >
      Full-time
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#848A96",
          },
          Object {
            "marginLeft": 5,
          },
        ]
      }
    >
      Â·
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#848A96",
          },
          Object {
            "fontSize": 13,
            "marginLeft": 5,
          },
        ]
      }
    >
      Remote
    </Text>
  </View>
</View>
`;
